var/spack/repos/builtin/packages/blat/package.py

```diff
@@ -23,4 +23,10 @@ def setup_build_environment(self, env):
     def install(self, spec, prefix):
         filter_file('CC=.*', 'CC={0}'.format(spack_cc), 'inc/common.mk')
         mkdirp(prefix.bin)
-        make("BINDIR=%s" % prefix.bin)
+        if spec.satisfies('%gcc@10:'):
+            make(
+                "BINDIR=%s" % prefix.bin,
+                "CFLAGS=-fcommon",
+                )
+        else:
+            make("BINDIR=%s" % prefix.bin)
```

Comment:
Spack has a special way of defining flags, needed since users can also add their own. Please see `https://spack.readthedocs.io/en/latest/packaging_guide.html#compiler-flags` (or grep the builtin `package.py` for `flag_handler`) and restore the `install` method here.